miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/MisoAppListener.java

```diff
@@ -123,6 +127,22 @@ public void contextInitialized(ServletContextEvent event) {
     loadIssueTrackerManager(misoProperties, context);
   }
 
+  private void linkMapsDir(ServletContext application, String fileStoragePath) {
+    try {
+      Path target = Paths.get(fileStoragePath, "freezermaps");
+      if (!Files.exists(target)) {
+        Files.createDirectory(target);
+      }
+      Path link = Paths.get(application.getRealPath("/"), "freezermaps");
+      if (Files.exists(link)) {
+        Files.delete(link);
```

Comment:
Won't this delete any freezer maps from previous versions of miso when an update happens?

Reply:
It's deleting the symlink; not the actual directory/files